tests/ctst/features/azureArchive.feature

```diff
@@ -232,6 +232,35 @@ Feature: Azure Archive
         |               Versioned |           2 |      30000 |      10 |
         |               Suspended |           2 |      30000 |      10 |
 
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    @ColdStorage
+    @BucketNotification
+    Scenario Outline: Restore notifications are triggered
+    Given a "<versioningConfiguration>" bucket
+    And one notification destination
+    And i subscribe to "s3:ObjectRestore:*" notifications for destination 0
+    And a transition workflow to "e2e-azure-archive" location
+    And <objectCount> objects "obj" of size <objectSize> bytes
+    Then object "obj-1" should be "transitioned" and have the storage class "e2e-azure-archive"
+    And object "obj-2" should be "transitioned" and have the storage class "e2e-azure-archive"
+    When i restore object "obj-1" for <restoreDays> days
+    When i restore object "obj-2" for <restoreDays> days
```

Comment:
we restore 2 objects, but check notifcation for one only → one restore should be enough?